system/jhtree/ctfile.hpp

```diff
@@ -196,10 +207,11 @@ class jhtree_decl CNodeBase : public CWritableKeyNode
     inline offset_t getFpos() const { assertex(fpos); return fpos; }
     inline size32_t getKeyLen() const { return keyLen; }
     inline size32_t getNumKeys() const { return hdr.numKeys; }
-    inline bool isBlob() const { return hdr.leafFlag == 2; }
-    inline bool isMetadata() const { return hdr.leafFlag == 3; }
-    inline bool isBloom() const { return hdr.leafFlag == 4; }
-    inline bool isLeaf() const { return hdr.leafFlag != 0; }
+    inline bool isBlob() const { return hdr.leafFlag == NodeBlob; }
+    inline bool isMetadata() const { return hdr.leafFlag == NodeMeta; }
+    inline bool isBloom() const { return hdr.leafFlag == NodeBloom; }
+    inline bool isLeaf() const { return hdr.leafFlag != NodeBranch; }       // actually is-non-branch.  Use should be reviewed.
```

Comment:
Will you be reviewing use as part of this PR? And renaming?

Reply:
I am hesitant to include that change - I think I would prefer another jira. [HPCC-25618]